src/Core/AdminConsole/Models/Data/Organizations/OrganizationAbility.cs

```diff
@@ -21,6 +21,7 @@ public OrganizationAbility(Organization organization)
         UseResetPassword = organization.UseResetPassword;
         UseCustomPermissions = organization.UseCustomPermissions;
         UsePolicies = organization.UsePolicies;
+        HasOrganizationDomains = Utilities.StaticStore.GetPlan(organization.PlanType).HasOrganizationDomains;
```

Comment:
I forgot that plan features are replicated in the `Organization` table and tracked separately for each org. I never really understood why we did that, except to enable Custom plans, which we don't hand out anyway (to my knowledge).

I'll follow up to see what we should do here generally (do we need to add another column? Do we actually need these columns?) but for now I think this approach is OK, even though it's breaking the pattern a little.

My only request is to handle a `null` plan, it's possible (although unlikely) that an org has an invalid plan, and in that case I'd rather default to `false` than throw an exception.

Also, for consistent wording, `UseOrganizationDomains`.

Reply:
Good question on Custom plans. I looked up how we were gating Domain Verification on the client side and we just look at the `UseSso` field https://github.com/bitwarden/clients/blob/819c312ce272c4b33efbfdab9618ba090b294597/libs/common/src/admin-console/models/domain/organization.ts#L255

So I'm thinking maybe we don't even need a new field at all, just look at `UseSso`. What do you think?

Reply:
Domain Verification started out as an SSO feature - it makes the login flow smoother - but these new features are no longer related to SSO. You could use a verified domain to manage account deprovisioning better even without using SSO. So I think it's become its own feature.

But for now, given that we're not ready to introduce new columns over it, let's stick with the current pattern of using `UseSso` for this. I'll create a follow up ticket so we can come back and adjust later if required.